packages/table/src/Table.js

```diff
@@ -634,12 +624,7 @@ const Table = ({
   onScroll,
   searchText,
   width,
-  filteredIdList,
 }) => {
-  if (searchText !== '' && filteredIdList.size === 0) {
-    return <NoVariants width={width} height={height}>No variants found</NoVariants>
-  }
```

Comment:
Moved this to VariantTable. The base Table should be agnostic about what it is rendering.